docs-content/general/4_company/1_values.md

```diff
@@ -1,32 +1,29 @@
 ---
-title: Mission & Values
+title: Values
 slug: values
 createdAt: 2022-04-01T20:28:14.000Z
 updatedAt: 2022-04-15T02:07:22.000Z
 ---
 
-Our mission is to support developers (like you) to ship with confidence. We do this by giving you the tools you need to **uncover, resolve, and prevent** issues in your web app.
+## Overview
 
-With everything we build, we maintain an unwavering promise to use open source technologies so that you and your team don't have a dependency on our hosted offerings. 
-
-## We build in public.
+This doc acts as a reference for our values at highlight.io, which includes how we work and we build things. It acts a way for us to understand how we should operate and what the "startup mentality" means to us. If you'd like to learn more about our product philosophy (which is more related to our product) see [this doc](./product-philosophy.md).
 
-We strive to build in public in every way we can. This means sharing our roadmap, product specs, and company strategy. We see this as giving you all the more reason to consider joining us in [building highlight](https://careers.highlight.run).
+Our values below:
 
-## We prioritize cohesion.
+## We build in public.
 
-People may think that we're building multiple products (session replay, error monitoring, etc..) but we see it as one. Before we build anything new, we prioritize making it operate seemlessly with everything else e.g. see our [fullstack mapping guide](../../getting-started/2_frontend-backend-mapping.md).
+With everything we build, we maintain an unwavering promise to use open source technologies so that you and your team don't have a dependency on our hosted offerings. 
```

Comment:
This isn't orthogonal to the overview. Per what we chatted about, this should be about how we work as a team not a commitment to our customers.